solver/app/v2/types.go

```diff
@@ -23,8 +23,15 @@ type FillOriginData = bindings.ISolverNetFillOriginData
 type OrderID [32]byte
 
 type Order struct {
-	ID         OrderID
-	Resolved   OrderResolved
+	ID                 OrderID
+	FillInstruction    bindings.IERC7683FillInstruction
+	FillOriginData     []byte
+	DestinationSettler common.Address
+	DestinationChainID uint64
+	SourceChainID      uint64
+	MaxSpent           []bindings.IERC7683Output
+	MinReceived        []bindings.IERC7683Output
+
```

Comment:
Was easier to pull and validate everything we need from resolved when `newOrder`, instead of validating on read